hc/accounts/tests/test_switch_team.py

```diff
@@ -14,18 +14,24 @@ def test_it_switches(self):
         r = self.client.get(url, follow=True)
 
         ### Assert the contents of r
-
+        self.assertRedirects(r, "/checks/")
+        
 
     def test_it_checks_team_membership(self):
         self.client.login(username="charlie@example.org", password="password")
 
         url = "/accounts/switch_team/%s/" % self.alice.username
         r = self.client.get(url)
+
         ### Assert the expected error code
+        self.assertEqual(r.status_code, 403)
+        
 
     def test_it_switches_to_own_team(self):
         self.client.login(username="alice@example.org", password="password")
 
         url = "/accounts/switch_team/%s/" % self.alice.username
         r = self.client.get(url, follow=True)
         ### Assert the expected error code
+        self.assertRedirects(r, "/checks/")
+        
```

Comment:
Assert error code

Reply:
Status code 200 is being returned, cause Alice can actually switch on her team members.